docs/review.md

```diff
@@ -0,0 +1,259 @@
+# Introduction
+
+Welcome to the code review for the Todo Challenge!  Again, don't worry - you are not expected to have all the answers. The following is a code-review scaffold for Todo Challenge that you can follow if you want to.  These are common issues to look out for in this challenge - but you may decide to take your own route.
+
+If you can please use this form to tick off where your reviewee has successfully has successfully incorporated these guidelines!  This form helps us get an overall picture of how the whole cohort is doing - it's not an assessment of an individual student.
+
+# Step 0: Checkout and Run tests
+
+Please checkout your reviewee's code and run their tests. Read the code and try and use the app through the web interface.  You can also experiment with the engine in the console. How easy is it to understand the structure of their code? How readable is their code? Did you need to make any cognitive leaps to 'get it'?  Is it more complicated than it needs to be?
+
+# Step 1:  How far did they get?
+
+* Features
+  * [ ] Can store tasks
+  * [ ] Instant updates
+  * [ ] Marking tasks as done
+
+* Bonus Features
+  * [ ] Filters
+  * [ ] Display number of tasks
+  * [ ] Clearing completed tasks
+  * [ ] Deploy the app
+  * [ ] Persistence layer with Node
+  * [ ] CSS Awesomeness
+
+# Step 2: Structure and supporting files
+
+# Installation/Usage Instructions in README (+)
+
+As we have seen previously, the README is a great place to show the full story of how your app is used (from a user's perspective).  Include instructions for how to download and run the tests, run the app, e.g.:
+
+```sh
+$ git clone git@github.com:[USERNAME]/todo-challenge.git
+$ cd todo-challenge
+$ ???
+$ ???
+```
+
+To run app:
+
+```sh
+$ ???
+$ open ???
+```
+
+It's a great idea to include some screenshots in your README.  For more info on embedding images in a README: https://guides.github.com/features/mastering-markdown/
+
+e.g.:
+```
+![Screenshot](https://path_to_your_image)
+```
+
+You will need to host your images somewhere, e.g.:
+* http://imgur.com/
+* http://dropbox.com/
+
+
+## If using a framework, delete any unnecessary code
+
+## Clean up
+
+* Remove comments
+* Try to avoid having log statements in your production code
+
+## ensure package.json devDependencies lists bower as a dependency (++)
```

Comment:
Maybe something more along the lines of `Make sure the project setup is reproducible on a different machine. (i.e. running `npm install && bower install` sets up all the necessary dependencies`